### DISCUSSION TITLE
Submitted Wrong Synopsis with the other project

### DISCUSSION THREAD

**Discussion by saqibahmed13** [github/india#825] [General]
Dear sir/ma'am i have submitted my project along with the URL of one of the azure services which i have used, but it got rejected because earlier i had submitted the synopsis of other project ( I can submit synopsis only once as it has written below), Now i have a project along with the running link, i request you to allow me to resubmit my synopsis once again.

**Comment by ey-mentor1** [comment]
Please send the mail to FRT support team for updating the project synopsis.
Once it's updated with respect to working project, please submit after then.

**Comment by ey-mentor1** [comment]
Hello, 
You can modify the project synopsis on the Project Submission Tab by clicking on Edit.
Once you have done the required changes and added the GitHub and project URL,  you can click on submit. The edited details will be submitted.
Thank you

**Comment by saqibahmed13** [comment]
Thank you for your response,but there is no button showing me to submit
again

**Comment by ey-mentor1** [comment]
Hello,
Can you please check the expiry date for your ID.
Currently we are extending the expiry dates for students, you'll receive mail for that.
Post that you can edi, save and submit.
Thank you
